### PR TITLE
Update ansible, python and terraform tooling for the new release

### PR DESCRIPTION
**Pull request by LukasZahradnik**
Updates env variables, ansible version, terraform version, terraform scripts

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,15 +4,15 @@ services:
     build: .
     container_name: purestorage_fusion_lab
     environment:
-      - API_CLIENT=pure1:apikey:1234567891234567
+      - FUSION_ISSUER_ID=pure1:apikey:1234567891234567
```
Comment (DenDen039) [inline docker-compose.yml:7]: @mikenelson-io  Do we really need these docker compose file?

**Review by DenDen039 — APPROVED** [review]

**Review by dimleevik — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (andreepyro) [reply docker-compose.yml:7]: If you don't want to run long `docker run` commands including the environment setup etc., docker-compose file is the option. It makes the usage of the devkit easier.

**Review by andreepyro — APPROVED** [review]
LGTM

File: docker-compose.yml (hunk 1 above)
Comment (LukasZahradnik) [reply docker-compose.yml:7]: Can we even run it with docker-compose without any edit? There are some arguments that have to be specified

Comment (andreepyro) [reply docker-compose.yml:7]: I didn't really check. But it should match to what we have in readme.

Comment (LukasZahradnik) [reply docker-compose.yml:7]: docker-compose isn't mentioned in readme

Comment (andreepyro) [reply docker-compose.yml:7]: I mean, the parameters for the `docker run` [commands mentioned in the readme](https://github.com/PureStorage-OpenConnect/fusion-client-devkit/blob/main/README.md?plain=1#L80) should (hopefully) match with the docker-compose file.

Comment (LukasZahradnik) [reply docker-compose.yml:7]: I just don't know how to pass them via command line, if that's even possible.

Comment (mikenelson-io) [reply docker-compose.yml:7]: I do not believe that the compose file is necessary for this project. We supply the command lines, and while they are a bit lengthy, I don't see them as overly complex for users.

**Pull request merged**